package.json

```diff
@@ -35,8 +37,5 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  },
-  "devDependencies": {
```

Comment:
In our project we don't have to split dependencies between dev and not dev. We will talk about this difference later.